transforms/handle_the_DC.xsl

```diff
@@ -17,7 +17,7 @@
     <xsl:template match="dc:dc">
         <xsl:copy>
             <xsl:apply-templates/>
-            <identifier type="hdl">
+            <dc:identifier xmlns:dc="http://purl.org/dc/elements/1.1/" type="hdl">
```

Comment:
Prefix is [already defined in the stylesheet](https://github.com/discoverygarden/islandora_handle/pull/13/files#diff-61cb4fe92331890bd19753ca905d2b2fR5), shouldn've have to declare again.